### ISSUE TITLE
Deliberate CPKT - This file type is not supported.

### ISSUE THREAD

**Issue by Rashxz** [replicate/dreambooth#36]
Hi, Im tryng to train a model with replicate/dreambooth last version (a8ba568da0313951a6b311b43b1ea3bf9f2ef7b9fd97ed94cebd7ffd2da66654), Im using deliberate.cpkt as cpkt_base but when I start the process I got this error: "This file type is not supported.Try another that is plain text, audio, image, or video.". I got the deliberate.cpkt from civitai and hugging face. Am I doing something wrong or missing something? Any help would be really appreciated. 

Thanks.

**Comment by thesrs02** [comment]
No, you're doing everything right. The API for Dreambooth on replicate should work. That error message is just a bug in the web interface.

Hope this helps you.
